osquery/CMakeLists.txt

```diff
@@ -69,6 +69,10 @@ else()
   ADD_OSQUERY_LINK_CORE("glog")
 endif()
 
+ADD_OSQUERY_LINK_CORE("broker")
```

Comment:
You are right. I used these `FindX` when testing with shared libraries installed locally. I removed them